tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh

```diff
@@ -62,7 +62,7 @@ for prj in "${no_prjs[@]}"; do
 	log_mustnot zfs set projectquota@$prj=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even if that id is nonexistent"
```

Comment:
This is fine. One other idea would be "can set all numeric id even if that id does not exist"